### PR TITLE
Add how to cite in README.md

### PR DESCRIPTION
**Pull request by benoit-cty**
Following https://github.com/mlco2/codecarbon/discussions/550

### CODE REVIEW

**Review by sashavor — APPROVED** [review]
Perfect!
I just added my own name - it was missing for some reason :upside_down_face:

**Comment by vict0rsch** [comment]
I don't know why you think that I had something to do with this ommission @sashavor but yeah you should be there.

File: README.md [hunk 1]
```diff
@@ -120,6 +121,53 @@ Check out our [contribution guidelines :arrow_upper_right:](https://github.com/m
...
+@software{benoit_courty_2024_11171501,
+  author       = {Benoit Courty and
+                  Victor Schmidt and
+                  Sasha Luccioni and
+                  Goyal-Kamal and
+                  MarionCoutarel and
+                  Boris Feld and
+                  Jérémy Lecourt and
+                  LiamConnell and
+                  SabAmine and
```
Comment (SaboniAmine) [inline README.md:139]: You can remove this line.

**Comment by benoit-cty** [comment]
> Perfect! I just added my own name - it was missing for some reason 🙃

That's weird, but GitHub don't see any commit from your account : https://github.com/mlco2/codecarbon/graphs/contributors

But you certainly deserve to have your name in the list.

File: README.md (hunk 1 above)
Comment (benoit-cty) [reply README.md]: Thanks

**Comment by SaboniAmine** [comment]
I'm sorry, but please next time avoid using commit messages to transmit this kind of information.
We have efficient ways to solve communication issues, and a commit history is not one of them.

**Comment by sashavor** [comment]
I disagree, it's good to have transparency on these kinds of things, and
digital traces of messages sent and received I've got nothing to hide!

**Comment by SaboniAmine** [comment]
You are right, and this thread seems, at least to me, the correct channel to have this discussion. It saves all context of this discussion, and the list of commits & file additions enables a meaningful history. 
And indeed, the absence of your name could be @vict0rsch's intention, or it could be not. 
As Benoît explained, the initial list of contributors have been automatically generated & didn't include you, for a mysterious reason, but it doesn't seem related to Victor. I was also mentioned twice, confirming the buggy generation.
Oppositely, the history of messages in git have much less context, and is more prone to false interpretations. Here is what this message looks like in history, with few place to doubt. I would like to avoid this, and tend to more [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
image

**Pull request merged**